stdlib/2and3/warnings.pyi

```diff
@@ -1,33 +1,37 @@
 # Stubs for warnings
 
-# Based on http://docs.python.org/3.2/library/warnings.html
+from typing import Any, Dict, NamedTuple, Optional, TextIO, Union
+from types import ModuleType, TracebackType
 
-from typing import Any, List, TextIO, Union
-
-def warn(message: Union[str, Warning], category: type = ...,
+def warn(message: Union[str, Warning], category: Optional[type] = ...,
```

Comment:
Now that Type[C] exists we can describe the type of category more precisely: it must be `Optional[Type[Warning]]`.